packages/core/admin/admin/src/components/UnstableGuidedTour/Tours.tsx

```diff
@@ -55,26 +70,37 @@ export const GuidedTourOverlay = styled(Box)`
   inset: 0;
   background-color: rgba(50, 50, 77, 0.2);
   z-index: 10;
-  pointer-events: none;
 `;
 
 const UnstableGuidedTourTooltip = ({
   children,
   content,
   tourName,
   step,
+  requiredActions,
 }: {
   children: React.ReactNode;
   content: Content;
   tourName: ValidTourName;
   step: number;
+  requiredActions?: GetGuidedTourMeta.Response['data']['completedActions'];
 }) => {
+  const { data: guidedTourMeta } = useGetGuidedTourMetaQuery();
```

Comment:
I think we can move forward with this here but I did just realize that eventually we will want to determine the current step of each tour based on the completedActions